### DISCUSSION TITLE
WCIU-DT2 (The U Inpendent)

### DISCUSSION THREAD

**Discussion by stanloonabelike** [iptv-org/community#788] [Q&A]
### Channel Name

The U

### Channel ID

WCIUDT2.us

### Website

https://www.wciu.com/

### Notes

I can't find it anywhere. 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19793
